md/release-notes.md

```diff
@@ -1,18 +1,143 @@
+* Start jobs: start a job on Orchestrator on a specified number of Robots. (compatible with _job input parameters_  in version 2018.3)
+* Add queue item: add a new item in a queue in order to exchange data with jobs
+* Get job status: retrieve the current status of a job
+Moreover, UiPath provides native activities to interact with Bonita in order to start a Case, send a BPM Message, validate a Task. (see UiPath documentation for more details)
+
+<a id="project-explorer"/>
+
+### Project explorer in Bonita Studio
+In one glance, view the content of the current Bonita project. In Bonita 7.8, the project is the equivalent of what used to be a repository. Now repository is dedicated to SVN and Git, where Bonita becomes only about projects. 
+From this view, you can create, open, edit, delete and use SVN and Git commands, simply with a right click.
+**Diagrams:**
+The BPMN palette is now included in the diagram view, and if you collapse it, it still automatically opens when your mouse passes over its closed panel, and closes once you have selected the element to drag and drop on the diagram.
+The overview of the diagram is now stacked in the properties view, after the Validation tab, and is now called "Minimap".
+As for the Tree view, it is displayed just by the Project explorer, as a deep dive in the diagram content.
+**REST API extensions:**
+The tree view is now nested in the global project explorer.
+
+<a id="callback"/>
+
+### Bonita callback method
+A new REST API sendMessage is available for when Bonita has launched an external service (like UiPath or DocuSign) and needs to be called back in an asynchronous manner.
+
+<a id="modal"/>
+
+### Modal container to create pop-ins 
+A new container is now available in the UI Designer palette. To manage such containers, the button widget gains two actions: "Open modal" and "Close modal".
+It is important to notice that on the editor, embedding a modal window in another modal window is not supported. Embed the buttons to open pop-ins in pop-ins, but not the modal containers themselves.
+
+<a id="workers-logs"/>
+
+### Monitoring of number and details of engine workers
+Busy engine workers and their activity is now available through JMS counters. Plug a JMS console on Bonita JVM to monitor workers activity.
+
+<a id="convert"/>
+
+### Change the type of a UI Designer artifact, from page to form and from form to page
```

Comment:
This sentence is hard to read :laughing: !

Reply:
Changed it to "Change the type of a UI Designer artifact (page/form)"